src/main/java/teammates/ui/webapi/action/CreateCourseAction.java

```diff
@@ -27,8 +28,11 @@ public void checkSpecificAccessControl() {
     @Override
     public ActionResult execute() {
         String newCourseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
```

Comment:
`COURSE_ID` is not something to identify the resource and therefore it should inside `CourseCreateRequest`